tests/run-all.sh

```diff
@@ -1,7 +1,9 @@
 #! /bin/bash
-set -eu
+set -u
 
-E2E_REGEX=${E2E_TEST_REGEX:-*.test.ts}
+# TODO - Remove TypeScript regex after all tests have been migrated to Go.
+E2E_REGEX_GO="${E2E_TEST_REGEX}_*test.go"
+E2E_REGEX_TS="${E2E_TEST_REGEX}-*test.ts"
```

Comment:
```suggestion
E2E_REGEX_GO=${E2E_TEST_REGEX:-*_test.go}
E2E_REGEX_TS=${E2E_TEST_REGEX:-*.test.ts}
```
Should be probably somethig like this, to prevent unbound variable problem.